transport_udp.go

```diff
@@ -32,7 +32,9 @@ import (
 // in the batch, because the length of the list is encoded as varint32, as well as SeqId.
 const emitBatchOverhead = 30
 
-var errSpanTooLarge = errors.New("Span is too large")
+var errSpanTooLarge = errors.New("span is too large")
+
+const maxInt64 = ^uint64(0) >> 1
```

Comment:
What is the reason for bit-shifting? We would lose half of the available sequence numbers, which is quite a lot (2^63)!
If the concern is overflowing the `uint64` value, we could modify `addInt64` to something like:
```
func addInt64(v *uint64, delta uint64) { // Changed delta to uint64 to simplify logic
        if *v > maxUint64 - delta { // prevent overflow
		*v -= maxUint64 - delta + 1 // if we want to start at 0
	} else {
		*v += delta
	}
}
```

Could we use the built-in golang `math` constant which reduces the surface area for mistakes or confusion? i.e. (unfortunately, the `math` constants are untyped so it is interpreted as `int` by default hence can't be used directly)?
```
const maxUint64 uint64 = math.MaxUint64
```

nit: consider co-locating consts together, though not necessarily need to be grouped.

Reply:
will use math.MaxInt64